DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/CentreConfigurationController.cs

```diff
@@ -80,13 +90,36 @@ public IActionResult EditCentreWebsiteDetails(EditCentreWebsiteDetailsViewModel
                 return View(model);
             }
 
+            model.CentrePostcode = model.CentrePostcode!.Trim();
+            var mapsResponse = mapsApiHelper.GetPostcodeCoordinates(model.CentrePostcode).Result;
+
+            if (mapsResponse.Status == "ZERO_RESULTS")
+            {
+                ModelState.AddModelError(nameof(model.CentrePostcode), "Enter a valid postcode.");

```

Comment:
I think the new consistent style of error messages is without a full stop in the end (but do double check on latest master just in case) 🙏